modules/terraform-aci-access-monitoring-policy/terraform-aci-dhcp-relay-policy/.terraform-docs.yml

```diff
@@ -0,0 +1,34 @@
+version: '>= 0.14.0'
```

Comment:
something is wrong with that module folder, i think you copied dhcp relay folder inside monitoring policy